src/components/NearMe/Wrappers/Cards/NearMeCard.js

```diff
@@ -6,39 +6,39 @@ import mapPinIcon from "../../icons/map-pin.svg";
 
 export default function NearMeCard(props) {
     return (
-        <div class="bg-white w-full flex items-center p-2  shadow border-2 border-gray-400">
-            <div class="flex items-center space-x-4">
+        <div className="bg-white w-full flex items-center p-2  shadow border-2 border-gray-400">
```

Comment:
className seems to have deprecated! @athul

Reply:
Is it deprecated in ReactJS  too? Also ES lint suggested to change to className.

Reply:
@sreenadh I don't think so. Maybe React class based Components are deprecated but not ClassNames, They're the equivalent of HTML class attribute in React right?

Correct me if I'm wrong

Reply:
Yea, even you get an error in console if you use `class=""`  instead of `className=""` . Since React uses JS and class is a keyword in JS language, this causes ambiguity.  

> React uses JS, `class` is a keyword in javascript and JSX is an extension of javascript. That's the principal reason why React uses `className` instead of `class`.
Source : [Stackoverflow - class vs className in React 16](https://stackoverflow.com/questions/46989454/class-vs-classname-in-react-16#:~:text=class%20is%20a%20keyword%20in,has%20changed%20in%20that%20regard.&text=Token%20class%20denotes%20that%20the,follows%20is%20a%20class%20declaration.)